### ISSUE TITLE
Handling empty/non-existent directories

### ISSUE THREAD

**Issue by aescott** [lukasgeiter/mkdocs-awesome-pages-plugin#24]
Hey, great plugin! I wrote my own plugin to conditionally include some .md files according to custom criteria (using the `on_files` event). Sometimes this logic ends up excluding an entire folder's contents, but the folder is still listed in my "arrange:" config in my `.pages` for the parent folder, so awesome-pages barfs when it parses my `.pages`.

The easiest way to fix this, but it's hacky, is to modify the .pages in my plugin temporarily, then restore it post-build. But that's ugly.

Any better way of ignoring empty folders when building the nav tree?

**Comment by lukasgeiter** [comment]
You can turn the error into a warning by setting the `strict` option to `false` ([see example in readme](https://github.com/lukasgeiter/mkdocs-awesome-pages-plugin#options)).

Beyond that, do you think there is a need for some way to turn the warning off entirely?

**Comment by aescott** [comment]
No i think that'd be fine. Could add an option to ignore empty folders, but not sure that fits in the simplicity of this thing?

**Comment by aescott** [comment]
Tried that and it still puts the folder name in, with a "none" sub item: 
----
![image](https://user-images.githubusercontent.com/1957589/92041485-3105db00-ed2d-11ea-8fd5-c569640d4daf.png)
---

I also noticed that my plugin (which overrides on_files) and yours don't play well together. With my on_files override, it falls back to alphabetical ordering... which is odd because awesome-pages only overrides on_nav. Tried putting them in different orders in my config, didn't make a difference. 

Just picked this up yesterday, not sure if I'm missing something obvious.
